dali/base/dafdesc.cpp

```diff
@@ -3506,6 +3506,7 @@ class CStoragePlaneInfo : public CInterfaceOf<IStoragePlane>
     virtual const char * queryHosts() const override { return xml->queryProp("@hosts"); }
```

Comment:
queryPrefix() probably needs reimplementing - it does not include the optional @subPath.  Possibly something that should be done for 8.4.  Created HPCC-26518.